pkg/registries/adapters/local_openshift_adapter.go

```diff
@@ -90,18 +90,32 @@ func (r LocalOpenShiftAdapter) FetchSpecs(imageNames []string) ([]*apb.Spec, err
 			r.Log.Errorf("Failed to load image spec")
 			continue
 		}
-		if strings.HasPrefix(image.Name, registryIP) {
-			// Image has proper registry IP prefix
-			spec.Image = image.Name
-			namespace := strings.Split(image.Name, "/")[1]
-			for _, ns := range r.Config.Namespaces {
-				if ns == namespace {
-					r.Log.Debugf("Image [%v] is in configured namespace [%v]. Adding to SpecList.", image.Name, ns)
-					specList = append(specList, spec)
-				}
-			}
+		if strings.HasPrefix(image.Name, registryIP) == false {
+			r.Log.Debugf("Image does not have a registry IP as prefix. This might cause problems but not erroring out.")
```

Comment:
Why not `Warningf` then?

Reply:
@djzager Because if you look a few lines below it is possible that S2I images will not have any registry prefix at all. I'm indifferent to which type of logging statement it is but with 10 S2I images that would output 10 warning statements in the logs for something that is expected behavior.

Feels weird to warn the user over expected behavior.

Reply:
I figured there was a reason. In this case, the text of the message is the warning. Works for me.